sdk/clientcore/core/pom.xml

```diff
@@ -77,6 +78,13 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.conscrypt</groupId>
+      <artifactId>conscrypt-openjdk</artifactId>
```

Comment:
I'd really rather not ship with this as a compile dependency. I'd rather offer guidance on using this to improve performance. This library hasn't shipped in a long time and if any CVEs against it come up we may be stuck.